clouddriver-ecs/src/main/java/com/netflix/spinnaker/clouddriver/ecs/model/loadbalancer/EcsLoadBalancer.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+ * CONDITIONS OF ANY KIND, either express or implied. See the License
+ * for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package com.netflix.spinnaker.clouddriver.ecs.model.loadbalancer;
+
+import com.amazonaws.services.elasticloadbalancingv2.model.Listener;
+import com.netflix.spinnaker.clouddriver.ecs.EcsCloudProvider;
+import com.netflix.spinnaker.clouddriver.model.LoadBalancer;
+import com.netflix.spinnaker.clouddriver.model.LoadBalancerServerGroup;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+import lombok.Data;
+
+@Data
+public class EcsLoadBalancer implements LoadBalancer {
```

Comment:
The EcsLoadBalancer and the EcsLoadBalancerCache are similar (scheme and targetGroupServices exist in one vs the other). In what scenarios will you use one over the other (EcsLoadBalancerCache vs EcsLoadBalancer)?

Reply:
`EcsLoadBalancerCache` is the load balancer data we expect to be cached by clouddriver, while `EcsLoadBalancer` is the data we want to be returned (to deck, mostly) about ECS load balancers in an app. 

The latter should include additional things we need to populate the LB view, (e.g. targetGroupServices) and I expect it to diverge further over time as we add things like instance counts and task health which we want to be available to the load balancers view but isn't the responsibility of the caching agent to keep track of.

Reply:
Got it. Would it make sense for one to extend the other or for them to be their own concrete implementations?

Reply:
hmm you know it would probably be better to have `EcsLoadBalancer` extend `'cache`. At first I was thinking the data type of `serverGroups` might have to be different (maybe include add'l target health info) but it's part of the parent `LoadBalancer` interface so we wouldn't want to change it anyway. +1

Reply:
per offline discussion: going to add a todo re: extending and look into refactoring `EcsLoadBalancerCache` in a separate PR.